2022-08-02 23:45:02,980 ERROR: Exception on /login [GET] [in d:\github\microblog2\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\microblog2\app\routes.py", line 72, in login
    return render_template('login.html', title='Sign In', form=form)
  File "d:\github\microblog2\lib\site-packages\flask\templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "d:\github\microblog2\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\microblog2\app\templates\login.html", line 26, in template
    <a href="{{ url_for{'reset_password_request') }}">Click to Reset It</a>
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '{'
2022-08-02 23:45:13,326 INFO: Microblog Startup [in D:\microblog2\app\__init__.py:46]
2022-08-02 23:45:14,483 ERROR: Exception on /login [GET] [in d:\github\microblog2\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\microblog2\app\routes.py", line 72, in login
    return render_template('login.html', title='Sign In', form=form)
  File "d:\github\microblog2\lib\site-packages\flask\templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "d:\github\microblog2\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\microblog2\app\templates\login.html", line 26, in template
    <a href="{{ url_for{'reset_password_request') }}">Click to Reset It</a>
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '{'
2022-08-02 23:45:16,966 ERROR: Exception on /login [GET] [in d:\github\microblog2\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\microblog2\app\routes.py", line 72, in login
    return render_template('login.html', title='Sign In', form=form)
  File "d:\github\microblog2\lib\site-packages\flask\templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "d:\github\microblog2\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\microblog2\app\templates\login.html", line 26, in template
    <a href="{{ url_for{'reset_password_request') }}">Click to Reset It</a>
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '{'
2022-08-02 23:45:17,973 ERROR: Exception on /login [GET] [in d:\github\microblog2\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 2077, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1525, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1523, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\github\microblog2\lib\site-packages\flask\app.py", line 1509, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "D:\microblog2\app\routes.py", line 72, in login
    return render_template('login.html', title='Sign In', form=form)
  File "d:\github\microblog2\lib\site-packages\flask\templating.py", line 149, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "d:\github\microblog2\lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 768, in compile
    self.handle_exception(source=source_hint)
  File "d:\github\microblog2\lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "D:\microblog2\app\templates\login.html", line 26, in template
    <a href="{{ url_for{'reset_password_request') }}">Click to Reset It</a>
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '{'
